egs/wham/ConvTasNet/eval.py

```diff
@@ -1,14 +1,16 @@
+#!/usr/bin/env python3
 import os
 import random
+from pprint import pprint
+
+import argparse
 import soundfile as sf
 import torch
 import yaml
 import json
-import argparse
 import numpy as np
 import pandas as pd
 from tqdm import tqdm
-from pprint import pprint
```

Comment:
Why changing imports?

Reply:
Oh I don't know what happened here. Will fix. 

But actually I will change them later if necessary. This is more like a style practice: for imports, usually we put built-in imports first, libraries then and functions from other scripts at last.

Reply:
IMO, either we use isort which does it automatically (we could), or we mainly what you say, which is mainly the case here. 
It's not worth too much energy. 
Maybe `isort` would make sense, so it cuts the debate short